docker/packager/binary/Dockerfile

```diff
@@ -3,6 +3,53 @@
 ARG FROM_TAG=latest
 FROM clickhouse/test-util:$FROM_TAG
 
+# Rust toolchain and libraries
+ENV RUSTUP_HOME=/rust/rustup
```

Comment:
Why do we need rust toolchain?

Reply:
Well, I took the latest Dockerfile from upstream. Maybe we can omit this, but let's first check it really works

Reply:
Removed